mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -147,8 +151,10 @@
     <string name="preferences_sync_bookmarks">Bookmarks</string>
     <!-- Preference for signing out -->
     <string name="preferences_sign_out">Sign out</string>
+    <!-- Preference displays and allows changing current FxA device name -->
+    <string name="preferences_sync_device_name">Device name</string>
     <!-- Label indicating that sync is in progress -->
-    <string name="sync_syncing">Syncing</string>
+    <string name="sync_syncing">Syncing…</string>
```

Comment:
This will need a string ID change so that localizers see they need to update this string